qutip/solver/heom/bofin_solvers.py

```diff
@@ -602,9 +613,9 @@ class HEOMSolver(Solver):
         "state_data_type": "dense",
     }
 
-    def __init__(self, H, bath, max_depth, *, options=None):
+    def __init__(self, H, bath, max_depth, odd_parity=False, *, options=None):
```

Comment:
```suggestion
    def __init__(self, H, bath, max_depth, *, odd_parity=False, options=None):
```
Let's make the `odd_parity` argument keyword only too.